eng/Versions.props

```diff
@@ -91,6 +92,7 @@
     <MicrosoftNuGetBuildTasksVersion>0.1.0</MicrosoftNuGetBuildTasksVersion>
     <MicrosoftPortableTargetsVersion>0.1.2-dev</MicrosoftPortableTargetsVersion>
     <MicrosoftServiceHubClientVersion>2.0.48</MicrosoftServiceHubClientVersion>
+    <MicrosoftServiceHubFrameworkVersion>2.0.84</MicrosoftServiceHubFrameworkVersion>
```

Comment:
Should this be kept in sync with MicrosoftServiceHubClientVersion? (Not sure if they are "related" versions or not...)

Reply:
I don't think it's necessary, and there are binding redirects in VS for both anyway.

Reply:
You should link these two. Hubclient and Framework are changing some in tandem and you may suffer regressions if they are out of sync for your tests at runtime.